orangecontrib/snom/widgets/owpreprocessimage.py

```diff
@@ -185,9 +197,39 @@ def image_values_fixed_levels(self):
     def __init__(self):
         self.markings_list = []
         super().__init__()
+        SelectionMaskImageOpts2DMixin.__init__(self)
 
         self.preview_runner = ImagePreviewRunner(self)
 
+        mbox = gui.widgetBox(self.controlArea, "Mask selection")
```

Comment:
This box should be moved above the "Commit" button. "Commit" button should be almost always at the bottom.

"Mask selection" -> "Mask" or "Masking"

And then, use the design language from above. So before the variable selector, make a label, perhaps "Column" and "Value" fro the value.